shared/src/main/scala/squants/unitgroups/ImplicitDimensions.scala

```diff
@@ -0,0 +1,104 @@
+package squants.unitgroups
+
+import squants.Dimension
+import squants.electro._
+import squants.energy._
+import squants.information.{DataRate, Information}
+import squants.mass.{AreaDensity, ChemicalAmount, Density, Mass}
+import squants.motion._
+import squants.photo._
+import squants.radio._
+import squants.space._
+import squants.thermal.{Temperature, ThermalCapacity}
+import squants.time.{Frequency, Time}
+
+/**
+  * Puts [[Dimension]] objects into implicit scope
+  * so that implicit [[UnitGroup]] machinery works.
+  */
+object ImplicitDimensions {
```

Comment:
No, there wouldn't be. You'll get a compiler error if you try to use it, but this file will build just fine. Which is a problem. I'm not sure the best way to address it.